cmd/imagec/errors.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+// DoNotRetry is an error wrapper indicating that the error cannot be resolved with a retry.
+type DoNotRetry struct {
+	Err error
```

Comment:
Because only the exported members of Embedded types are exported. You cannot do what you suggest as compiler will throw an error like ```...DoNotRetry field 'error' in struct literal```. You will need a NewDoNoRetry function to achieve what you want but since you are trying to save a function this won't work as it will introduce one.